pyproject.toml

```
@@ -33,6 +33,7 @@ dependencies = [
   "grpcio >= 1.54.2, < 2",
   "grpcio-tools >= 1.54.2, < 2",
   "protobuf >= 4.25.3, < 6",
+  "frequenz-client-base >=0.5",
```

The [Release Notes](https://github.com/frequenz-floss/frequenz-client-base-python/releases/tag/v0.4.0) mention that one should use 
```suggestion
  "frequenz-client-base[grpcio] >= 0.5.0, < 0.6.0",
```